src/main/java/core/basesyntax/strategy/BalanceOperation.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements Operation {
+    private final FruitStorage fruitStorage;
+
+    public BalanceOperation(FruitStorage fruitStorage) {
+        this.fruitStorage = fruitStorage;
+    }
+
+    @Override
+    public void performOperation(FruitTransaction transaction, FruitStorage fruitStorage) {

```

Comment:
We have FruitStorage class field
```suggestion
    public void performOperation(FruitTransaction transaction) {
```